editor/action_map_editor.cpp

```diff
@@ -58,7 +58,15 @@ void ActionMapEditor::_event_config_confirmed() {
 	Array events = new_action["events"].duplicate();
 
 	if (current_action_event_index == -1) {
-		// Add new event
+		// Add new event if not already bound
+		for (const Variant& event : events)
```

Comment:
You should run clang format / the post-commit hook on your code, this would fail the CI pipeline.

- Opening brace of for-loop and if-block must be on same line.
- Newly added comments should end with a period.